13/umbraco-cms/tutorials/multilanguage-setup.md

```diff
@@ -198,6 +198,38 @@ To render dictionary items in the template, replace the text with the following
 @Umbraco.GetDictionaryValue("Contact Number")
 ```
 
+### Translating Multi-Value Property Editors
+
+When building a multilingual website in Umbraco, you can use property editors like Dropdown, Checkbox List, and Radiobutton List to offer content in multiple languages. By leveraging dictionary items, you can ensure that these property editors display the appropriate translated values based on the site's current language.
```

Comment:
I think this first sentence makes it sound like ONLY these three property editors can offer content in multiple languages 🙈 

Perhaps something like this instead:

```suggestion
Property editors like Dropdown, Checkbox List, and Radiobutton List let you define a set of prevalues to be chosen between in the Content section. By leveraging dictionary items, you can ensure that these prevalues are translated correctly based on the site's current language.
```